apps/lend/src/components/PageMarketList/Page.tsx

```diff
@@ -34,16 +35,30 @@ const Page: NextPage = () => {
 
   const isLoadingApi = useStore((state) => state.isLoadingApi)
   const updateTableRowsSettings = useStore((state) => state.marketList.updateTableRowsSettings)
+  const setStateByKey = useStore((state) => state.marketList.setStateByKey)
 
   const [loaded, setLoaded] = useState(false)
   const [parsedSearchParams, setParsedSearchParams] = useState<SearchParams | null>(null)
 
   const { signerAddress } = api ?? {}
 
-  const FILTER_MAPPER: FilterMapper = {
-    all: { id: 'all', displayName: t`All` },
-    user: { id: 'user', displayName: t`My markets` },
-  }
+  const FILTER_MAPPER: FilterMapper = signerAddress
```

Comment:
It might be worth creating 2 constants outside of this component, and using only them in this condition
```
const SIGNER_FILTER_MAPPER: FilterMapper = {...} as const
const DEFAULT_FILTER_MAPPER: FilterMapper = {...} as const
```

```suggestion
  const FILTER_MAPPER = signerAddress ? SIGNER_FILTER_MAPPER : DEFAULT_FILTER_MAPPER
```